src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public void getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
```suggestion
public class Lottery {
    public void getRandomBall() {
        Ball ball = new Ball();
        ball.setNumber(random.nextInt(100));
```
create Random and ColorSupplier field in class Lottery

Reply:
@aaananas 
OK, I did it. But why is this solution better? Is it bad to create Random or instances of other classes right away in a method?

Reply:
Just for incapsulation?

Reply:
It is to not create new instance of Random class every time we call getRandomBall() method